libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMapInstanceState.java

```diff
@@ -30,8 +33,8 @@ public String retrieveWayname() {
     return waynameText;
   }
 
-  public boolean isCameraTracking() {
-    return isCameraTracking;
+  public MapPaddingInstanceState retrieveMapPadding() {
```

Comment:
> Access can be package-private

The same thing happens with `isWaynameVisible`, `retrieveWayname` and `retrieveMapPadding` methods in `NavigationMapboxMapInstanceState`.